libs/data-mapper/src/lib/utils/__test__/DataMapUtils.spec.ts

```diff
@@ -163,6 +164,18 @@ describe('utils/DataMap', () => {
       expect(resultEntries[3][1]).toBeTruthy();
     });
 
+    it('creates a simple conditional connection', () => {
+      simpleMap['ns0:Root'] = {
+        DataTranslation: {
+          EmployeeName: {
+            RegularFullTime: '$if(is-greater-than(/ns0:Root/ConditionalMapping/ItemPrice, /ns0:Root/ConditionalMapping/ItemQuantity))',
+          },
+        },
+      };
+      const result = convertFromMapDefinition(simpleMap, extendedSource, extendedTarget, [greaterThanFunction, conditionalFunction]);
```

Comment:
Instead of importing the individual functions, you could just use "functionMock" it has all of the functions we use in standalone